docs/tables/aws_redshift_cluster.md

```diff
@@ -60,3 +57,13 @@ from
 where
   not encrypted;
 ```
+
+### List clusters audit logging status
+
+```sql
+select
+  cluster_identifier,
+  logging_details -> 'LoggingEnabled' as LoggingEnabled
+from
+  aws_redshift_cluster;
```

Comment:
Can we update this example to list logging status for a specific cluster, instead of listing all clusters?